themes/mongodb/src/js/componentThirdParty.js

```diff
@@ -28,15 +12,16 @@ export function setup(fastNav) {
     const branch = document.body.getAttribute('data-branch');
 
     try {
-        const sampleFactor = SAMPLE_FACTORS[project] || 0.1;
-        window.delighted.survey({
-            minTimeOnPage: 180,
-            sampleFactor: sampleFactor,
-            properties: {
-              project,
-              branch,
-            }
-        });
+        if (ENABLED_SITES_FOR_DELIGHTED.has(project)) {
+            const projectName = project === 'docs' ? 'manual' : project;
+            window.delighted.survey({
+                minTimeOnPage: 90,
+                properties: {
+                    branch,
+                    project: projectName,
+                }
+            });
+        }
```

Comment:
Does anything here restrict surveys from rendering on our staging environment (https://docs-mongodbcom-staging.corp.mongodb.com/) ?

Reply:
Actually, I just saw your comment in the main PR description. If there is no easy way on legacy, we can punt on that for now and re-evaluate if it becomes a major problem.

Reply:
Is there a way to retrieve the URL in that part of the code and maybe check for something like the above? Regardless, I'm still good to punt.

Reply:
Yeah, we could probably just check for the staging url in a case like this. I'll make the change and try it out

Reply:
Much appreciated, if it turns out to be a time sink, let me know and we'll punt.